addon-test-support/audit-if.js

```diff
@@ -0,0 +1,31 @@
+import RSVP from 'rsvp';
+import a11yAudit from './audit';
+import utils from './audit-if-utils';
+
+/**
+ * A method to return the value of queryParameter
+ *
+ * @method getUrlParameter
+ * @private
+ */
+function getUrlParameter(name) {
+    name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
```

Comment:
Can we remove this? Since we know that this function isn't going to be used for all kinds of `name`s, I think this is unnecessary.